Source/WebCore/html/canvas/CanvasRenderingContext.cpp

```diff
@@ -151,10 +153,7 @@ bool CanvasRenderingContext::wouldTaintOrigin(const HTMLVideoElement* video)
     if (!video || !m_canvas.originClean())
         return false;
 
-    if (!video->hasSingleSecurityOrigin())
```

Comment:
Can we remove HTMLVideoElement:: hasSingleSecurityOrigin() and related, or do you plan to do it in a follow-up?

Reply:
I will take it in a follow-up